eternalcore-core/src/main/java/com/eternalcode/core/feature/ignore/IgnoreServiceImpl.java

```diff
@@ -0,0 +1,81 @@
+package com.eternalcode.core.feature.ignore;
+
+import com.eternalcode.core.event.EventCaller;
+import com.eternalcode.core.feature.ignore.event.IgnoreAllEvent;
+import com.eternalcode.core.feature.ignore.event.IgnoreEvent;
+import com.eternalcode.core.feature.ignore.event.UnIgnoreAllEvent;
+import com.eternalcode.core.feature.ignore.event.UnIgnoreEvent;
+import com.eternalcode.core.injector.annotations.Inject;
+import com.eternalcode.core.injector.annotations.component.Service;
+import org.bukkit.Server;
+import org.bukkit.entity.Player;
+
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+
+@Service
+public class IgnoreServiceImpl implements IgnoreService {
```

Comment:
Son, in this class, everything in IntelliJ IDEA is yellow. Are you sure you haven't missed anything?